discord_connector.py

```diff
@@ -0,0 +1,346 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# -----------------------------------------
+# Phantom sample App Connector python file
+# -----------------------------------------
+
+# Phantom App imports
+import phantom.app as phantom
+from phantom.action_result import ActionResult
+from phantom.base_connector import BaseConnector
+
+# Usage of the consts file is recommended
+# from discord2_consts import *
+import requests
+import json
+from bs4 import BeautifulSoup
+import discord
+import asyncio
+
+
+class RetVal(tuple):
+
+    def __new__(cls, val1, val2=None):
+        return tuple.__new__(RetVal, (val1, val2))
+
+
+class Discord2Connector(BaseConnector):
+
+    def __init__(self):
+
+        # Call the BaseConnectors init first
+        super(Discord2Connector, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```
At this point there's no need to stick to the pre py3 syntax ;)